app/retail/views.py

```diff
@@ -16,13 +16,17 @@
     along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 '''
+
+
 import logging
 from json import loads as json_parse
+from json import load as json_load
+from json import dump as json_dump
```

Comment:
none of these seem used?

Reply:
Oh, It is used by previous version and is useless now. Just delete it.